test/functional/dd-school-data-spec.js

```diff
@@ -48,4 +48,24 @@ fdescribe( 'The dynamic financial aid disclosure', function() {
      expect( page.jobRate.getText() ).toEqual( '18' );
   } );
 
+  it( 'should dynamically display the graduation cohort content if it\'s available', function() {
+    browser.get( 'http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/offer/?iped=234492&pid=94&oid=740280139f3238cbc9702c7b0d62e5c238a835d0&book=850&gib=6000&gpl=0&hous=20000&insl=0&insi=1&inst=1&mta=4500&othg=2000&othr=500&parl=0&pelg=4000&perl=0&prvf=0&prvi=8&prvl=0&ppl=0&schg=2000&stag=2000&subl=1000&totl=88592&tran=5000&tuit=30000&unsl=1000&wkst=0' );
+     browser.sleep( 600 );
+     page.confirmVerification();
+     browser.sleep( 1000 );
+     page.continueStep2();
+     browser.sleep( 1000 );
+     expect( page.graduationCohortContent.isDisplayed() ).toBeTruthy();
```

Comment:
Minor: prefer `.toBe( true )` to `toBeTruthy()` (see http://stackoverflow.com/questions/32615713/tobetrue-vs-tobetruthy-vs-tobetrue)